index.html

```diff
@@ -466,6 +470,9 @@ <h3 id="sec-processing-model"><span class="secno">4.4 </span>Processing Model</h
                 <p>Fire a simple event named <code>visibilitychange</code> that bubbles, isn't cancelable, and has no default action,
                 at the <a href="http://www.w3.org/TR/html5/dom.html#document">Document</a>.</p>
             </li>
+            <li>
+              <p>Run <a href="#document-is-visible">document is visible</a>.</p>

```

Comment:
You can't run, "document is visible" as it's a concept, not an algorithm. Maybe there needs to be a `[[visible]] slot` that is set at various points. So, a document is **visible** if the slot is set to true. A document can only be set to visible if X conditions are met.